.github/workflows/go.yaml

```diff
@@ -57,7 +57,13 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        tinygo-version: ['0.34.0'] # FIXME: this needs to be manually bumped; something should ping when a new version is available
+        go-version: ["1.22", "1.23", "1.24"]
+        tinygo-version: ["0.34.0", "0.35.0", "0.36.0", "0.37.0"] # FIXME: this needs to be manually bumped; something should ping when a new version is available
+        exclude:
+          - go-version: "1.24"
+            tinygo-version: "0.35.0"
```

Comment:
Can these follow the same order as above?

Reply:
done